src/cmd/src/options.rs

```diff
@@ -38,7 +38,7 @@ pub struct MixOptions {
     pub frontend: FrontendOptions,
     pub datanode: DatanodeOptions,
     pub logging: LoggingOptions,
-    pub wal_meta: MetaSrvWalConfig,
+    pub wal_meta: MetasrvWalConfig,
```

Comment:
I remember we use `MetaSrv` prefix anywhere. Would you have any suggestions @fengjiachun

Reply:
‌‌‌‌Yes, it is recommended to use `MetaSrv`.

Reply:
Pardon me, which one of `Metasrv` and `MetaSrv` should be used? @fengjiachun

Reply:
> A search in our codebase shows that we use "MetaSrv" and "Metasrv" both. I think we should unify to "Metasrv" because it can be a noun that refers specifically to our meta server. Same reason we "Datanode" instead of "DataNode".

It seems that all type definitions are using `MetaSrv`. I suggest continuing to use `MetaSrv` for now. If it needs to be changed to Metasrv, please address it in the next PR and ensure compatibility.

Reply:
Good.